### DISCUSSION TITLE
How to skip "Post Checkout repository" step

### DISCUSSION THREAD

**Discussion by fieldsfarmer** [actions/checkout#1347] [Q&A]
We use checkout action in our CI workflow which runs in our own runners. The CI will generate a lot of files and make the repository size pretty large. We want to keep the repo size small so it will also keep the runner's size small and I add a step to remove the repo at the end of the workflow. However this will fail the "Post Checkout repository" step which will always run after my last step in the workflow. Is there anyway to skip the "Post Checkout repository" step or make it as "continue-on-error"? Thanks!

**Comment by Spiritaine** [comment]
Continue on

**Reply by fieldsfarmer** [reply]
> Continue on

What do you mean?

**Comment by chandanrattan** [comment]
Even i do have same concern. Checkout on every Job will lead to performance issues if someone has Big Size repo
